priv/test_repo/migrations/1_migrate_all.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Absinthe.Ecto.TestRepo.Migrations.MigrateAll do
```

Comment:
I'd recommend keeping all migrations in a single file, since it's easier to edit/see the resulting schema in a single file, and the repo is dropped every test run anyway.